examples/data-sources/git_repository/data_source_cluster.tf

```diff
@@ -0,0 +1,14 @@
+# Read Tanzu Mission Control git repository : fetch cluster git repository details
+resource "tanzu-mission-control_git_repository" "create_git_repository" {
```

Comment:
Please check the resource name, it should not be create, since this is data source. Differentiate with the name like- cluster_scoped_git_repository

Reply:
Updated PTAL